README.md

```diff
@@ -148,82 +155,83 @@ This feature is still under development, review, and testing.
 
 #### Status
-| Property         | Type     | Comments                                                                                                   |
+| Property         | Type     | Description                                                                                                   |
 |:-----------------|:---------|:-----------------------------------------------------------------------------------------------------------|
-| instanceID   | `string`   | The service instance ID in SAP Cloud Service Management service |
-| operationURL | `string`   |  URL of ongoing operation for the service instance |
-| operationType   |  `string`   |  The operation type (CREATE/UPDATE/DELETE) for ongoing operation |
-| conditions       | `[]condition`   |  An array of conditions describing the status of the service instance. <br>The possible condition types are:<br>- `Ready`: set to `true`  if the instance is ready and usable<br>- `Failed`: set to `true` when an operation on the service instance fails, in this case the error details are available in the condition message.
+| instanceID   | `string`   | The service instance ID in SAP Cloud Service Management service. |
+| operationURL | `string`   |  The URL of the current operation performed on the service instance. |
+| operationType   |  `string`   |  The type of the current operation. Valid values are CREATE, UPDATE, or DELETE.  |
+| conditions       | `[]condition`   |  An array of conditions describing the status of the service instance. <br>The possible condition types are:<br>- `Ready`: set to `true`  if the instance is ready and usable<br>- `Failed`: set to `true` when an operation on the service instance fails. In this case, the details about the error are available in the        condition message.
 
 
 
 ### Service Binding 
 #### Spec
-| Parameter             | Type       | Comments                                                                                                   |
+| Parameter             | Type       | Description                                                                                                   |
 |:-----------------|:---------|:-----------------------------------------------------------------------------------------------------------|
-| serviceInstanceName`*`   | `string`   | The Kubernetes name of the service instance to bind, should be in the namespace of the binding |
-| externalName       | `string`   |  The name for the service binding in SAP Cloud Service Management service, defaults to the binding `metadata.name` if not specified |
+| serviceInstanceName`*`   | `string`   | The Kubernetes name of the service instance to bind, should be in the namespace of the binding. |
+| externalName       | `string`   |  The name for the service binding,, defaults to the binding `metadata.name` if not specified |
```

Comment:
why did you remove the "in SAP Cloud Service Management? for the service instances it says "The name for the service instance in SAP BTP, defaults to the instance `metadata.name` if not specified.". 
I think it is important, as the binding already has a name in the k8s cluster (metadata.name), this name is what's going to show up in BTP (cockpit etc.).

Reply:
@eyalnathan do you think " SAP Cloud Service Management" is more accurate than "in SAP BTP"?
The consumer creates instance/binding in K8S, and then sees it in BTP (cockpit etc.), not in "SAP Cloud Service Management".

Reply:
Restored